.circleci/config.yml

```diff
@@ -39,27 +39,27 @@ test_settings: &test_settings
 version: 2
 jobs:
 
-  py38:
+  py310:
     <<: *test_settings
     docker:
-      - image: python:3.8-buster
+      - image: python:3.10-buster
```

Comment:
While we're switching images, I'd suggest switching to bookworm (or at least bullseye) -- buster is scheduled to end support 30 June 2024.

Reply:
Fixed!

Reply:
Actually I am seeing this error with codecov:

```
{'detail': ErrorDetail(string='Missing "owner" argument. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
```

Any idea what's going on there?